packages/flutter/lib/src/material/bottom_sheet.dart

```diff
@@ -72,7 +70,7 @@ class BottomSheet extends StatefulWidget {
   /// [ScaffoldState.showBottomSheet], for persistent bottom sheets, or by
   /// [showModalBottomSheet], for modal bottom sheets.
   const BottomSheet({
-    Key key,
+    Key? key,
     this.animationController,
```

Comment:
looks like this should be required

Reply:
But there are place where no animationController is passed. And sometimes a nullable value is passed as well.